### DISCUSSION TITLE
[Keyball39] 左右の特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by monishim** [Yowkees/keyball#573] [Q&A]
お世話になっております。Keyball39にて左右ともに特定のキーが反応しません。
具体的な箇所については添付画像をご覧頂ければと思います。

以下、実施した切り分けと対応についてです。
・ダイオード導通チェック
　→問題があると思われる箇所についてはダイオードの交換
・ProMicro導通チェック
　→特に問題なし

![IMG_4943](https://github.com/Yowkees/keyball/assets/122177/9d8868cd-74fe-4bf1-b30d-cf3d1789f9d9)
![IMG_4944](https://github.com/Yowkees/keyball/assets/122177/366ce720-78dd-47d2-98c8-e5946b77662d)

お忙しい中恐縮ですが、ご確認、アドバイス頂けると幸いです。
以上、宜しくお願い致します。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

不具合の起きているキーがバラバラとしていますので、キーソケットのはんだ付けを根気良くやり直してみてください。
横から見て、端子にはんだがしっかり馴染んでいるのを確認してください。

**Comment by Ryo624** [comment]
今回の反応のないキーに対応するダイオードは添付させていただいた写真のそれぞれの色どうしのものになります。
はんだの量が多すぎる箇所があったり、ダイオードが斜めを向いて上手くはんだ付けできていない箇所があるように見えますので、キーソケットのはんだ付けと併せてご確認ください。

![327944487-9d8868cd-74fe-4bf1-b30d-cf3d1789f9d9](https://github.com/Yowkees/keyball/assets/143698349/010e4802-d027-423f-ab4a-abd61101f48b)
![327944491-366ce720-78dd-47d2-98c8-e5946b77662d](https://github.com/Yowkees/keyball/assets/143698349/b5cf3262-e91b-46ff-915f-fae77ed9c167)

**Reply by monishim** [reply]
細かい情報ありがとうございます。
こちら頂いた情報をもとにダイオードとキーソケットの半田のなじませを実施したところすべてのキーが反応するようになりました。大変助かりました。ありがとうございました。

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます

ぜひ末永くご愛用いただければ幸いです。
